### ISSUE TITLE
Soundflower installation fails

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mattscotthudson/soundflower#78]
```
What steps will reproduce the problem?
1. Attempt to install Soundflower 
2.
3.

What is the expected output? What do you see instead?
Installation of Soundflower devices


What version of the product are you using? On what operating system?
1.5.1 Snow Leopard 10.6.4

Please provide any additional information below.
I had Airfoil installed, with the soundflower 'extra'. I removed soundflower 
via the 'extra' pane.
Installation of soundflower 1.5.1 fails. In the logs, I found this:

04-09-10 10:18:36   installd[247]   PackageKit: Install Failed: PKG: post-flight 
scripts for "com.cycling74.soundflower.pkg"
Error Domain=PKInstallErrorDomain Code=112 UserInfo=0x10041fb10 "An error 
occurred while running scripts from the package “soundflower.pkg”." {
    NSFilePath = "./postflight";
    NSLocalizedDescription = "An error occurred while running scripts from the package \U201csoundflower.pkg\U201d.";
    NSURL = "./Contents/Packages/soundflower.pkg -- file://localhost/Volumes/Soundflower-1.5.1/Soundflower.mpkg/";
    PKInstallPackageIdentifier = "com.cycling74.soundflower.pkg";
}

(more logs are attached)

Funny thing is that I am able to install SoundFlower via AirFoil. Anyone?
```

Original issue reported on code.google.com by `s.balken...@gmail.com` on 4 Sep 2010 at 8:40

Attachments:
- [Soundflower.log](https://storage.googleapis.com/google-code-attachments/soundflower/issue-78/comment-0/Soundflower.log)

**Comment by GoogleCodeExporter** [comment]
```
Issue 111 has been merged into this issue.
```

Original comment by `t...@electrotap.com` on 1 Feb 2012 at 9:15

**Comment by GoogleCodeExporter** [comment]
```
Issue 80 has been merged into this issue.
```

Original comment by `t...@electrotap.com` on 1 Feb 2012 at 9:17

**Comment by GoogleCodeExporter** [comment]
```
This should be fixed in the just-released version 1.6 release candidate.
```

Original comment by `t...@electrotap.com` on 3 Feb 2012 at 8:50
- Changed state: **Fixed**
